src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,181 @@
+package duke;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Task;
+
+
+/**
+ * User Interface that user directly interacts with
+ * e.g. displaying messages, scanning inputs
+ */
+public class Ui {
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String END =
+            "    ---------------------------------------------------------------------------------";
+
+    private static final String INDENTATION = "     ";
```

Comment:
Is there a reason for the indentation for each output line? I assume it is to improve readability?